Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM openjdk:8-jre-alpine
+FROM hmcts/cnp-java-base:openjdk-jre-8-alpine-1.4
 
-WORKDIR /opt/app
+# Mandatory!
+ENV APP pdf-service.jar
+ENV APPLICATION_TOTAL_MEMORY 1024M
+ENV APPLICATION_SIZE_ON_DISK_IN_MB 66
```

Comment:
is pdf service that big when you build it?

Reply:
docker image is 87mb - ill update 👍